pennylane/_grad.py

```diff
@@ -324,7 +324,16 @@ def _jacobian_function(*args, **kwargs):
                 "If this is unintended, please add trainable parameters via the "
                 "'requires_grad' attribute or 'argnum' keyword."
             )
-        jac = tuple(_jacobian(func, arg)(*args, **kwargs) for arg in _argnum)
+        try:
+            jac = tuple(_jacobian(func, arg)(*args, **kwargs) for arg in _argnum)
+        except Exception as e:
+            raise ValueError(
+                "PennyLane has a new return shape specification that"
+                " may not work well with autograd and more than one measurement. That may"
+                " be the source of the error. \n\n"
+                "Try wrapping your output in a autograd numpy array,"
```

Comment:
That is not working for different size measurement, I would suggest just to link the documentation.